fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/rdf/impl/SkolemNodeRdfContext.java

```diff
@@ -93,6 +94,10 @@ public SkolemNodeRdfContext(final FedoraResource resource,
             return null;
 
         } catch (final RepositoryException e) {
+            if (e instanceof ItemNotFoundException) {
+                // Effectively skip resources that do not exist
```

Comment:
That depends on how you define "intra-repository links". If you define them as links between to existing resources... then that precludes the notion of not resolving.
If you define them as links between an existing resource and one that does not exist... then, yes, that is exactly what this commit does.